src/ctap/command.rs

```diff
@@ -82,13 +83,13 @@ impl Command {
         let command_value = bytes[0];
         match command_value {
             Command::AUTHENTICATOR_MAKE_CREDENTIAL => {
-                let decoded_cbor = cbor::read(&bytes[1..])?;
+                let decoded_cbor = cbor_read(&bytes[1..])?;
```

Comment:
This works because of line 54. With the new wrapping function, maybe just integrate it into `cbor_read`?

Reply:
Done.